application/client/src/app/ui/elements/transport/details/serial/component.ts

```diff
@@ -16,8 +22,47 @@ export class TransportSerial extends ChangesDetector {
     @Input() public source!: SerialTransportSettings;
     @Input() public session!: Session;
 
+    @ViewChild('stdin') public input!: AutocompleteInput;
+
```

Comment:
id = "stdin" was for commands as soon as it was sending to `stdin` on child process )
It isn't an error, but just could confuse developer (even you in a couple of months)